easybuild/easyblocks/l/llvm.py

```diff
@@ -1366,7 +1371,7 @@ def _sanity_check_dynamic_linker(self):
             for suffix in ('.c', '.o', '.x'):
                 remove_file(f'{test_fn}{suffix}')
 
-    def sanity_check_step(self, custom_paths=None, custom_commands=None, *args, **kwargs):
+    def sanity_check_step(self, *args, custom_paths=None, custom_commands=None, **kwargs):
```

Comment:
Since those are not forced to be keyword-only arguments in `framework` i do not think we should force them here

Reply:
Then `*args` should be removed or funny things might happen according to [PyLint](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/keyword-arg-before-vararg.html):

> When defining a keyword argument before variable positional arguments,
> one can end up in having multiple values passed for the aforementioned parameter
> in case the method is called with keyword arguments.

TBH: I don't think we need either at all. Other easyblocks don't have any params or just args&kwargs to pass through.

Reply:
I would probably fix this in a dedicated PR ...  accompanied to a framework one if needed

Reply:
Ok, just reverted this for now.